env_build_commands.md

```diff
@@ -2,14 +2,10 @@
 
 ```bash
 conda create -y -n dmri_seg python=3.9
```

Comment:
dmri_seg needs to be changed to match the version you are using to activate with: dmri-seg